models/issues/issue_watch.go

```diff
@@ -105,12 +105,9 @@ func GetIssueWatchers(ctx context.Context, issueID int64, listOptions db.ListOpt
 		And("`user`.prohibit_login = ?", false).
 		Join("INNER", "`user`", "`user`.id = `issue_watch`.user_id")
 
-	if listOptions.Page > 0 {
-		sess = db.SetSessionPagination(sess, &listOptions)
-		watches := make([]*IssueWatch, 0, listOptions.PageSize)
-		return watches, sess.Find(&watches)
-	}
-	watches := make([]*IssueWatch, 0, 8)
+	listOptions.SetDefaultValues()
```

Comment:
The new logic will force the pagination but the pagination could be ignored in the old logic.